src/mame/video/zaxxon.cpp

```diff
@@ -463,6 +463,18 @@ uint32_t zaxxon_state::screen_update_razmataz(screen_device &screen, bitmap_ind1
 	return 0;
 }
 
+uint32_t zaxxon_state::screen_update_ixion(screen_device &screen, bitmap_ind16 &bitmap, const rectangle &cliprect)
+{
+	draw_background(bitmap, cliprect, false);
+	// On Ixion the fg tilemap is used to blank out erased tiles, the sprites must appear above these
+	// whereas on Razzmatazz the fg tilemap is used to mask off sprites near the edges so must appear
+	// above them.
+	// Is there a missing priority control bit or is this a hardwired change on the PCB?
```

Comment:
The Zaxxon and Congo Bongo schematics don't seem to show anything controlling graphics priority besides transparency. It's more likely that Sega rewired the boards, though that would take substantial work since the palette and foreground are on the same board.

Reply:
hmm interesting, although it is based on razmataz, not zaxxon/congo bongo, and razmataz already has some changes to the tilemap handling.

I'll update the comment, but either way the game is basically unplayable with the current setup, you can't see enemy bullets half the time, which results in you being sniped by things you never saw.  Also visually it makes no sense because enemy explosions destroy tiles, but the explosion animations etc. are currently instantly hidden by the tile that is applied to the fg layer.

Reply:
I've changed the wording